facebook/docusaurus
# Localizes blog page dates using moment.js

This PR renders dates based on browser locale.

@rickyvetter requested this support on https://github.com/facebookexperimental/Docusaurus/issues/143

How to test:

1. Switch your machine's locale/language to Japanese (or another language).
2. Restart browser
3. Load a page from blog and note that the date is localized.

English US:
<img width="556" alt="screen shot 2017-10-24 at 7 13 56" src="https://user-images.githubusercontent.com/23874/31948192-1ac0b8bc-b88b-11e7-9eb4-80b16c75d5e1.png">

Japanese:
<img width="576" alt="screen shot 2017-10-24 at 7 14 28" src="https://user-images.githubusercontent.com/23874/31948198-1c89dd36-b88b-11e7-83e8-64d689fb0474.png">
